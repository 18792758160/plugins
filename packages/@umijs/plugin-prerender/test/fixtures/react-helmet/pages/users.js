/**
 * title: Users
 */
import React from 'react';
import { Helmet } from 'react-helmet';
const styles = {
  normal: 'normal___1KMnC',
};

function Page(props) {
  return (
    <div className={styles.normal}>
      <h1>Page users111</h1>
      <h2>users</h2>
      <Helmet>
        <title>Hello Umi SSR Helmet user title</title>
      </Helmet>
      <ul>
        {(props.list || []).map(user => (
          <li key={user}>{user}</li>
        ))}
      </ul>
    </div>
  );
}

Page.getInitialProps = () => {
  console.log('Users getInitialProps');
  return {
    list: ['foo', 'bar'],
  };
};

export default Page;
