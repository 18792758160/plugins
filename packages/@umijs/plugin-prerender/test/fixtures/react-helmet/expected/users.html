<!DOCTYPE html><html lang="en" data-reactroot><head><title data-react-helmet="true">React Helmet Users index</title><link rel="stylesheet" href="/"><meta charset="UTF-8"><meta name="viewport" content="width=device-width, initial-scale=1.0"><title>Default Title</title><script>window.g_useSSR=true;
  window.g_initialData = {"list":["foo","bar"]};</script><script>window.routerBase = "/";</script><script>window.UMI_PRERENDER_SUM_CODE = "7e0c0dd153f3601276478e09449c4608";</script><meta data-react-helmet="true" name="description" content="React Helmet title Users description"></head><body><div id="root"><div class="normal___1KMnC"><h1>Page users111</h1><h2>users</h2><ul><li>foo</li><li>bar</li></ul></div></div><script src="/umi.js"></script></body></html>